src/harness/util.py

```diff
@@ -285,3 +292,98 @@ def convertRequestToRequestWithCpiSignature(private_key, cpi_id,
   request['cpiSignatureData']['protectedHeader'] = jwt_message[0]
   request['cpiSignatureData']['encodedCpiSignedData'] = jwt_message[1]
   request['cpiSignatureData']['digitalSignature'] = jwt_message[2]
+
+class QueryPropagationAntennaModel:
+    nlcd_driver = nlcd.NlcdDriver()
+       
+    def getPPinPPA(self, boundarysPoints, ARCSEC):   
```

Comment:
Please use other name for the function for readability and  function convention. And also for the variable.
Add a docstring as well.
```  def GetProtectedPointsInPpa(boundary_points, res_arcsec):
         """Gets all the protected points in a PPA.
           Inputs:
              boundary_points: a list of (lon,lat) tuple defining the boundary of the points.
              res_arcsec: the gridding resolution (in arcsec).

           Returns:
             blah blah
          """
```
But even with that, this function is not correctly defined. A PPA cannot be easily defined by boundary_points, as it is generally a MultiPolygon with holes (ie several polygon possibly having holes).
We should redefine  this function to takes as input a GeoJSON (or shapely) defining the PPA zone, which supports all type of geometry, and not only a single polygon with no holes.

Other thing is that the function should compute the NLCD vote here, instead of letting the client code do it. If the function is moved to the PPA directory, that directory already instantiate the NLCD driver, so that can be used nicely, without having to instantiate a new one.

Basically to summarize we need to add a function in the PPA reference model directory to obtain the PPA gridding from a PPA definition. It will return the points and also the region code. 
I can eventually do that function quickly.